OpenRA.Utility/OpenRA.Utility.csproj

```diff
@@ -15,6 +16,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
     <Configurations>Release;Debug</Configurations>
+    <AssemblyName>Utility</AssemblyName>
```

Comment:
We can drop this and the Server assembly rename now that we have the bin dir right?